### PR TITLE
Issue/learn to code btn href

### PR DESCRIPTION
**Pull request by jamie-reece**
* **What does this PR introduce?** (Bug fix, feature, docs update, ...)

i've added `target="_blank"` to the ui buttons which link to freeCodeCamp so that they open in new tabs instead of redirecting away from our oxford page

### CODE REVIEW

**Comment by jamie-reece** [comment]
ok, i'll add those rel attributes tomorrow and resubmit the pr. thanks

File: index.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
         <a class="item" href="#aboutus">About Us</a>
         <a class="item" href="#ourpages">Members' pages</a>
         <div class="right item">
-          <a class="ui button" href="https://learn.freecodecamp.org">Learn to code</a>
+          <a class="ui button" href="https://learn.freecodecamp.org" target="_blank">Learn to code</a>
```
Comment (alanmynah) [inline index.html]: sorry, might have missed one :)

File: index.html [hunk 2]
```diff
@@ -34,7 +34,7 @@
     <a class="item" href="#ourpages">Members' pages</a>
     <div class="right menu">
       <div class="item">
-        <a class="ui button" href="https://learn.freecodecamp.org">Learn to code</a>
+        <a class="ui button" href="https://learn.freecodecamp.org" target="_blank" rel="noopener">Learn to code</a>
```
Comment (alanmynah) [inline index.html]: agree that we might not want to have "noreferrer".  so good job on this one

**Comment by jamie-reece** [comment]
sorry, not sure what happened there. i had some trouble adding commits to
that branch (i still haven't quite worked out how to work locally and push
to github from VS code - need to work on that.) thanks though

**Comment by alanmynah** [comment]
we can sort it out today in the evening

**Pull request merged**